src/javascript/JContent/actions/editImage/ImageEditorDialog/ImageEditorActions/ImageEditorActions.scss

```diff
@@ -0,0 +1,7 @@
+.expansePanel {
+    padding: 24px 24px 32px 0 !important;
+}
+
+.button {
+    margin-left: 0 !important;
```

Comment:
We should probably look at how we can set without using `!important` here. (we can usually override if we use with more specific selectors)

esp. this one as `.button` looks a bit more generic and not sure if that can affect other buttons

Reply:
Fixed with https://github.com/Jahia/jcontent/pull/536/commits/12ef5999fdfdeec3027d00dd55e061197788190d